pkgs/top-level/all-packages.nix

```
@@ -9437,6 +9437,8 @@ with pkgs;
   ipinfo = callPackage ../tools/networking/ipinfo { };
 
   ipscan = callPackage ../tools/security/ipscan { };
+  # `ipscan` is commonly known under the name `angryipscanner`
```

```suggestion
  # ipscan is commonly known under the name angryipscanner
```
I wouldn't set those in comments